scripts/rv_expression_association/prepare_inputs.py

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env python3
+
+import subprocess
+import sys
+
+subprocess.run([sys.executable, '-m', 'pip', 'install', 'scanpy==1.9.1'], check=True)  # 1.7.3?
+subprocess.run([sys.executable, '-m', 'pip', 'install', 'limix==3.0.4'], check=True)
+subprocess.run([sys.executable, '-m', 'pip', 'install', 'pandas_plink==2.2.9'], check=True)
+
+import logging
+import pandas as pd
+import scanpy as sc
+import xarray as xr
+# from cloudpathlib import AnyPath
+from cpg_utils import to_path
+from pandas_plink import read_plink1_bin
+from limix.qc import quantile_gaussianize
+
+# use logging to print statements, display at info level
+logging.basicConfig(format='%(levelname)s:%(message)s', level=logging.INFO)
+
+# sample_mapping_file = 'gs://cpg-tob-wgs-test/v0/skat/smf_Bcells.csv'
+phenotype_file = 'gs://cpg-tob-wgs-test/v0/skat/sce22.h5ad'
```

Comment:
We prefer to parameterise inputs (as opposed to constants defined in the top) in the function body with `click` for the command-line interface.

Reply:
on it!